include/wlr/types/wlr_xdg_shell_v6.h

```diff
@@ -98,9 +113,16 @@ struct wlr_xdg_popup_v6 {
 	struct wlr_seat *seat;
 	struct wlr_box geometry;
 
-	enum wlr_positioner_v6_anchor anchor;
-	enum wlr_positioner_v6_gravity gravity;
-	enum wlr_positioner_v6_constraint_adjustment constraint_adjustment;
+	struct wlr_xdg_positioner_v6_attributes positioner;
+	struct wlr_box anchor_rect;
```

Comment:
Is there a reason we have this twice? Once in the attributes, once on the wlr_xdg_popup itself.

Same for size and offset